### ISSUE TITLE
Rotation Gesture Detector Only works with Single ImageView.

### ISSUE THREAD

**Issue by quarks3k** [Almeros/android-gesture-detectors#5]
If I created Two ImageView instances in Single Activity and Attach Separate Instances of Rotation Gesture Detector to them the Only Rotation Listener for First ImageView Is Working Second Image's Rotation Gestures are not being able to detect .Please provide some solution to that its been 8 hours I am searching for solution.

Thanks.

**Comment by gmavenis** [comment]
I have more images on 1 Activity, but I can only touch on top image.
If I set as below, it's still not working:
android:layout_width="wrap_content"
android:layout_height="wrap_content"

I dont want process if touch outside of image. Please help.
Thanks.

**Comment by jpjodoin** [comment]
@gmavenis You can try my solution, it works for me. Let me know.

**Comment by AndroidDeveloperLB** [comment]
I'm having an issue with Rotation Gesture Detector, that it's quite hard to rotate the imageView, even if there is only a single one. 
what should I do to make it easier to rotate?

**Comment by AndroidDeveloperLB** [comment]
Anyway, I've made a fork that has this issue fixed:
https://github.com/AndroidDeveloperLB/android-gesture-detectors

**Comment by airtel121** [comment]
Still not worked for more then one images

**Comment by AndroidDeveloperLB** [comment]
@airtel121 Have you tried my fork ?

**Comment by airtel121** [comment]
Yes i replace TwoFingerGestureDetector.java from your fork.But only upper image is sensing the touch i set width and height to fill_parent so.
But if i set to wrap_content then image only scale inside its default location.

**Comment by airtel121** [comment]
How to archive functionality for example if i have two image inside FrameLayout having width and height set to fill_parent.  
-If i touch 1st image it should drag,rotate,zoom inside whole screen and if i touch second image it should drag,rotate,zoom inside whole screen.
Note:two image is just for sample purpose there may be multiple image inside FrameLayout and can be added dynamically.

**Comment by AndroidDeveloperLB** [comment]
First, you should avoid setting them both to fill_parent. Use wrap_content or a fixed sized. 
You can check out my sample here:
https://github.com/AndroidDeveloperLB/MultiTouchPlaceholderView

Not exactly what you want, but you can modify it to your needs.

**Comment by mariopce** [comment]
Fixed and tested on my fork
